XCUITests/NewTabSettings.swift

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+class NewTabSettingsTest: BaseTestCase {
+    func testCheckNewTabSettingsByDefault() {
+        navigator.goto(NewTabSettings)
+        waitforExistence(app.navigationBars["New Tab Settings"])
+        waitforExistence(app.tables.cells.staticTexts["Show your Top Sites"])
+        XCTAssertTrue(app.tables.switches["ASPocketStoriesVisible"].isEnabled)
+        XCTAssertTrue(app.tables.switches["ASBookmarkHighlightsVisible"].isEnabled)
+        XCTAssertTrue(app.tables.switches["ASRecentHighlightsVisible"].isEnabled)
+    }
+
+    func testToggleOffOnAdditionalContentBookmarks() {
+        // Bookmark one site and check it appears in a new tab
+        navigator.performAction(Action.BookmarkThreeDots)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        waitforExistence(app.staticTexts["Highlights"])
+
+        // Disable toggle and check that it does not appear in a new tab
+        navigator.goto(NewTabSettings)
+        navigator.toggleOff(userState.bookmarksInNewTab, withAction: Action.ToggleBookmarksInNewTab)
+        navigator.goto(HomePanelsScreen)
+        navigator.goto(NewTabScreen)
+
+        //This appears under top sites
+        waitforNoExistence(app.staticTexts["Highlights"])
+
+        // Enable toggle again and check it is shown
+        navigator.goto(NewTabSettings)
+        navigator.toggleOn(userState.bookmarksInNewTab, withAction: Action.ToggleBookmarksInNewTab)
+        navigator.goto(HomePanelsScreen)
+        navigator.goto(NewTabScreen)
+        navigator.nowAt(NewTabScreen)
```

Comment:
This doesn't make sense (`goto` followed by a `nowAt`)

Reply:
You are right, I removed that line